src/Model/Order/Order.php

```diff
@@ -131,11 +140,28 @@ public static function create(
 
     private function applyOrderWasCreated(OrderWasCreated $event): void
     {
-        $this->id              = $event->getId();
-        $this->businessName    = $event->getBusinessName();
+        $this->id = $event->getId();
+        $this->businessName = $event->getBusinessName();
         $this->businessAddress = $event->getBusinessAddress();
-        $this->contactPerson   = $event->getContactPerson();
-        $this->isPaid          = $event->getHasPaid();
-        $this->amount          = $event->getAmount();
+        $this->contactPerson = $event->getContactPerson();
+        $this->isPaid = $event->getIsPaid();
+        $this->amount = $event->getAmount();
+    }
```

Comment:
After this event is applied you will end up with an order that doesn't have a value for `$this->datePaid` - but it's non-nullable.

Reply:
Yeah, I thought about it, but I too had a second thought that the aggregate will be composed with the rest of the VO without datePaid. Ok cool I will change it to DatePaid.